README.md

```diff
@@ -26,6 +26,18 @@ Read more about Xash3D on ModDB: https://www.moddb.com/engines/xash3d-engine
 * Voice support
 * Vulkan renderer
 
+## Installation & Running
+0) Download Xash3D binaries: you can use [release](https://github.com/FWGS/xash3d-fwgs/releases) build (recommended) or [testing](https://github.com/FWGS/xash3d-deploy/tree/anewengine-master) build, also you can compile engine from sources.
```

Comment:
Those links are available at the top of README. 

I don't like repeating information but maybe it can be useful for people who don't know how to read.

I also think, we should not recommended using stable release at this point, it's too old.

Reply:
>I also think, we should not recommended using stable release at this point, it's too old.
Also thinking about it, but decided to keep both